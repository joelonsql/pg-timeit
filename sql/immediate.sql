BEGIN;

CREATE EXTENSION IF NOT EXISTS timeit;

SELECT timeit.s('pg_sleep', ARRAY['0.1']);
SELECT timeit.h('pg_sleep', ARRAY['0.1']);

--
-- Request two significant figures in result.
--
SELECT timeit.h('pg_sleep', ARRAY['0.1'], 2);

ROLLBACK;
