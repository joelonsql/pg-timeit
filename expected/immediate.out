BEGIN;
CREATE EXTENSION IF NOT EXISTS timeit;
SELECT timeit.s('pg_sleep', ARRAY['0.1']);
  s  
-----
 0.1
(1 row)

SELECT timeit.h('pg_sleep', ARRAY['0.1']);
   h    
--------
 100 ms
(1 row)

--
-- Request two significant figures in result.
--
SELECT timeit.h('pg_sleep', ARRAY['0.1'], 2);
   h    
--------
 100 ms
(1 row)

ROLLBACK;
